rubrik_cloudout_consolidation.template

```diff
@@ -0,0 +1,511 @@
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
+  "Description": "Complete the AWS configuration process required for the Rubrik CloudOut with consolidation. CloudOut uses an S3 bucket for archiving to Amazon. When consolidation for Amazon S3 is enabled, the snapshot consolidation runs in the AWS cloud using the cloud compute resources.",
+  "Metadata": {
+    "AWS::CloudFormation::Interface": {
+      "ParameterGroups": [
+        {
+          "Label": { "default": "Storage Configuration" },
+          "Parameters": ["CreateNewS3Bucket", "S3BucketName"]
+        },
+        {
+          "Label": { "default": "Network Configuration" },
+          "Parameters": ["VPC",
+            "Subnet",
+            "OnPremRubrikCIDR",
```

Comment:
With the pervasiveness of Cloud Clusters, using the term `OnPremRubrikCDIR` is confusing. Instead, we should make this more generic to any type of Rubrik Cluster. Something like `RubrikClusterCIDR` would work.

Reply:
Fixed by commit 3af27fc421ebfba6ca5832749afb830ab238b065